src/java.base/share/classes/java/lang/reflect/Array.java

```diff
@@ -49,13 +49,11 @@ private Array() {}
     /**
      * Creates a new array with the specified component type and
      * length.
+     * @implSpec
      * Invoking this method is equivalent to creating an array
      * as follows:
      * <blockquote>
-     * <pre>
-     * int[] x = {length};
-     * Array.newInstance(componentType, x);
-     * </pre>
+     * {@code Array.newInstance(componentType, new int[]{length});}
```

Comment:
Snippet is not just for cut-and-paste; it can also save us a `<blockquote>` tag as well, and syntax highlight support in IDEs.

Reply:
With the concession that more of the `<blockquote><pre>...</pre></blockquote>` blocks still in the JDK should be converted to snippets, I don't think that *all* of them should necessarily be converted.

Having done a few snippet conversions myself (JDK-8287838, JDK-8289399, JDK-8289775, JDK-8308987), I don't think this particular one is justified as a snippet.